test_scripts/Defects/8_1/3838_1_SDL_does_not_send_UI.ClosePopUp_in_case_VR.PerformInteraction_TIME_OUT.lua

```diff
@@ -0,0 +1,175 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/3838
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL does not send UI.ClosePopUp to HMI in case:
+-- - mobile App sends PerformInteraction request with VR_ONLY interaction mode to SDL
+-- - HMI responds with 'TIMED_OUT' to VR.PerformInteraction request
+--
+-- Pre-conditions:
+-- 1. SDL, HMI, Mobile session are started
+-- 2. App is registered and activated
+-- 3. CreateInteractionChoiceSet is added
+-- Steps:
+-- 1. App sends PerformInteraction request with VR_ONLY interaction mode to SDL
+-- 2. HMI responds with 'SUCCESS' to UI.PerformInteraction request
+-- 3. HMI responds with 'TIMED_OUT' to VR.PerformInteraction request
+-- SDL does:
+-- - not send UI.ClosePopUp request to HMI
+-- - send PerformInteraction response (resultCode: TIMED_OUT, success:false) to mobile App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local function getPromptValue(pText)
+  return {
+    {
+      text = pText,
+      type = "TEXT"
+    }
+  }
+end
+
+local initialPromptValue = getPromptValue(" Make your choice ")
+
+local helpPromptValue = getPromptValue(" Help Prompt ")
+
+local timeoutPromptValue = getPromptValue(" Time out ")
+
+local vrHelpvalue = {
+  {
+    text = " New VRHelp ",
+    position = 1
+  }
+}
+
+local requestParams = {
+  initialText = "StartPerformInteraction",
+  initialPrompt = initialPromptValue,
+  interactionMode = "BOTH",
```

Comment:
```suggestion
  interactionMode = "VR_ONLY",
```
Should just set this value correctly to begin with